### DISCUSSION TITLE
How to switch to the refactoring branch?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#459] [Q&A] [answered]
Do I have to install another version of Gamehub or can i do that in Gamehub itself.
Cause in the AUR I only found Gamehub and Gamehub-git.

**Comment by neuromancer** [comment]
If you want to test `refactoring` branch in ArchLinux:

1. Download the gamehub-git [PKGBUILD file](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=gamehub-git).
2. Replace the branch name in the `source` line (`dev` -> `refactoring`).
3. Run `makepkg`
4. Install the resulting .pkg.

**Comment by tkashkin** [comment, marked as answer]
It is in development and it's not really usable now.
You need to manually build it, see @neuromancer's answer how to do it on Arch.
